Apps/Examples/Examples/All Examples/CameraAnimatorsExample.swift

```diff
@@ -0,0 +1,75 @@
+import UIKit
+import MapboxMaps
+
+@objc(CameraAnimatorsExample)
+
+public class CameraAnimatorsExample: UIViewController, ExampleProtocol {
+
+    internal var mapView: MapView!
+
+    // Store the CameraAnimators so that the do not fall out of scope.
+    var bearingAnimator: CameraAnimator?
+    var centerAnimator: CameraAnimator?
+    var zoomAnimator: CameraAnimator?
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+
+        guard let accessToken = AccountManager.shared.accessToken else {
+            fatalError("Access token not set")
+        }
+
+        let resourceOptions = ResourceOptions(accessToken: accessToken)
+        mapView = MapView(with: view.bounds, resourceOptions: resourceOptions)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        view.addSubview(mapView)
+
+        mapView.centerCoordinate = CLLocationCoordinate2D(latitude: 37.7749, longitude: -122.4194)
+        mapView.zoom = 5
+
+        // Allows the delegate to receive information about map events.
+        mapView.on(.mapLoaded) { [weak self] _ in
+
+            guard let self = self else { return }
+
+            // Rotate the map 180º over the course of four seconds.
+            self.bearingAnimator = self.mapView.cameraManager.makeCameraAnimator(duration: 4,
+                                                                                    curve: .linear,
+                                                                           animationOwner: .custom(id: "bearing-animator"),
+                                                                               animations: {
+                self.mapView.bearing = 180.0
+            })
+            self.bearingAnimator?.addCompletion({ _ in
+                self.bearingAnimator = nil
```

Comment:
any reason to do this?

Reply:
I wasn't sure if the animator is released on its own once the animation is complete. Can remove!